anx/provider/loadbalancer/reconciliation/reconciliation.go

```diff
@@ -64,24 +64,12 @@ type reconciliation struct {
 
 	tags []string
```

Comment:
let's reduce this to just be a `string`, here and in `state_retriever` (and wherever else it's done this way)